### ISSUE TITLE
[Issue] No information on turning points, consequences for missing deployments or how to avoid breaching contract

### ISSUE THREAD

**Issue by Scoppio** [MegaMek/mekhq#6861]
### Brief Description *

A few years ago (I think between 2017 and 2021) I was playing MekHQ and I recall losing one battle and this causing a breach of contract, I had lost most of my mechs, had no money, I think I had to pay a giant fine for breaching contract, it basically destroyed my campaign as there was no way to get "pick me up contracts" to rebuild. I also missed one deployment once because all my mechs were being repaired and it also caused a breach of contract.

Now, we have support points, which isn't very clear to me how they are earned unless I read a doc, there are turning point scenarios and non turning point scenarios and I dont really know the difference between them, I've seen that now I can miss a deployment and it doesn't end the game anymore for me, but I'm really not grasping what exactly is the meaning or intent of all of this.

I would be very happy if there was a way to receive in-game a kind of "tutorial dialog" that would explain to me in as few words as possible the intent and how the game scenarios roll, and it may be dismissed "forever" automatically after the first time it's shown to me.

### 3. Steps to Reproduce

- There are no steps -

### Severity *

High (Major Disruption): A major feature is broken or incorrect, but a workaround exists.

### MekHQ Suite Version *

50.06

### Operating System *

Mac

### Java Version *

GraalVM - OpenJDK 21

### Final Verification

- [x] I confirm this is a single, unique issue that hasn't been reported before
- [x] I have filled and provided all necessary information above
- [x] I have included any and all logs, custom units, and CPNX (save) files
- [x] I have asked on MegaMek Discord about this issue
- [x] I have confirmed this issue is being opened on the correct repository: MegaMek, MegaMekLab, or MekHQ

**Comment by IllianiBird** [comment]
Has this been resolved by the addition of the tutorial text?

**Comment by Scoppio** [comment]
Yes
